components/style/rule_tree/mod.rs

```diff
@@ -412,14 +453,19 @@ impl StrongRuleNode {
         unsafe { &*self.ptr }
     }
 
+    /// Get the style source corresponding to this rule node. May return `None`
+    /// if it's the root node, which means that the node hasn't matched any
+    /// rules.
     pub fn style_source(&self) -> Option<&StyleSource> {
         self.get().source.as_ref()
     }
 
+    /// Get the importance that this rule node represents.
     pub fn importance(&self) -> Importance {
         self.get().importance
     }
 
+    /// Get an iterator for this rule node and its ancestors including.
```

Comment:
"including" is dangling here. Should maybe be removed or changed to "inclusive."